backend/src/v5/models/groups.constants.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Can we rename this to  metadata.rules.constants.js

The operators are on the rules afterall 😆